app/src/test/java/com/fcaputo/parkingapp/modelTests/ReservationModelTest.kt

```diff
@@ -0,0 +1,239 @@
+package com.fcaputo.parkingapp.modelTests
```

Comment:
a package should never be in camel case. It should be just model as you are in the tests